common/tools/dev-tool/src/util/samples/generation.ts

```diff
@@ -94,7 +94,9 @@ export async function makeSampleGenerationInfo(
 
   const sampleConfiguration = getSampleConfiguration(projectInfo.packageJson);
 
-  const baseName = projectInfo.name.split("/").slice(-1)[0];
+  const packageNameParts = projectInfo.name.split("/");
```

Comment:
Nit this could be combined into a single line `const [ scope, baseName ] = projectInfo.name.split("/");`